src/core.ts

```diff
@@ -631,13 +616,13 @@ export class NodePart implements Part {
   }
 
   private _setTemplateResult(value: TemplateResult): void {
-    const template = this.instance._getTemplate(value);
+    const template = this.templateFactory(value);
     let instance: TemplateInstance;
     if (this._value && this._value.template === template) {
       instance = this._value;
     } else {
-      instance = new TemplateInstance(
-          template, this.instance._partCallback, this.instance._getTemplate);
+      instance =
+          new TemplateInstance(template, value.processor, this.templateFactory);
```

Comment:
Maybe add a comment about using the `processor` from the `value` since this was an issue before?

Reply:
ack